internal/connor/connor.go

```diff
@@ -62,7 +66,7 @@ func matchWith(op string, conditions, data any) (bool, error) {
 		return anyOp(conditions, data)
 	case AllOp:
 		return all(conditions, data)
-	case EqualOp:
+	case EqualOp, request.FilterOpAlias:
```

Comment:
suggestion: I think it's fine to duplicate the `const` for `_alias` here and avoid the added dependency.

Reply:
done